editions/tw5.com/tiddlers/filters/syntax/Filter Parameter.tid

```diff
@@ -29,6 +29,8 @@ The parameter to a [[filter operator|Filter Operators]] can be:
 :: The parameter is the current value of the [[variable|Variables]] whose name appears between the angle brackets. Macro parameters are <<.em not>> supported up to v5.2.0
 ::<<.from-version "5.2.0">> Literal macro parameters are supported. For example: `[<now [UTC]YYYY0MM0DD0hh0mm0ssXXX>]`.
 
+Note: every [[filter Operator]] must be followed by a parameter expression. In the case of [[Operators without parameters]], that expression is empty, as with the filter Operator <<.olink links>> in `[<currentTiddler>links[]]`.
+
 ---
```

Comment:
Notes have a doc-macro. See: https://tiddlywiki.com/#Documentation%20Macros

Reply:
OK, I've fixed this issue as well... I see there have been lots of small problems with my PR suggestions, and when I fix one thing, sometimes I learn there is some other problem. If my suggestions are more trouble than they are worth, I will stop.